docs/io/visualization/convergence_plot.ipynb

```diff
@@ -0,0 +1,846 @@
+{
```

Comment:
<p>Highlight the fact that they are stored in <code>cplots</code>  attribute of simulation object <code>sim</code> . And also phrase it out that we have two plots - plasma and t_inner &amp; luminosites within it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1636/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
@atharva-2001 I believe only this is left?

Reply:
I guess I fixed this in https://github.com/tardis-sn/tardis/pull/1636/commits/4401372fd30d2c9846db172d9f546e200e26eca0
Could you please check if it's up to the mark? 
[documentation](https://atharva-2001.github.io/tardis/branch/convergence_plots/io/visualization/convergence_plot)